src/injected/components/details-dialog.tsx

```diff
@@ -200,7 +201,7 @@ export class DetailsDialog extends React.Component<DetailsDialogProps, DetailsDi
     }
 
     private renderInspectMessage(): JSX.Element {
-        if (this.props.dialogHandler.isInspectButtonDisabled(this)) {
+        if (this.props.dialogHandler.shouldShowInspectButtonMessage(this)) {
             return (
                 <div className="insights-dialog-inspect-disabled">
                     {`To enable the Inspect HTML button, open the developer tools (${this.props.devToolsShortcut}).`}
```

Comment:
Seems like this text should be updated (since the button is actually enabled all the time).
Maybe change to "To **use** the Inspect HTML button., **first** open..."?

Reply:
updated.